packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -87,13 +114,39 @@ export class Tabs extends React.Component<TabsProps & OUIAProps, TabsState> {
       // most recently selected tabContent
       tabContentRef.current.hidden = false;
     }
+    // Update scroll button state and which button to highlight
+    setTimeout(() => {
+      this.handleScrollButtons();
+    }, 1);
```

Comment:
Using `setTimeout` is kind of a red flag. Is there something we can do via `componentDidUpdate` instead, when state has changed?

Better yet, perhaps use the callback provided by `setState`?

Reply:
Why is this needed?

Reply:
This was preexisting.  Not sure why it is here.  I am looking into it.

Reply:
@jschuler this is what I was asking about.  I do not understand why we need this.

Reply:
good question, i did that a year ago :D let's remove it